docs/quickstart/basics.rst

```diff
@@ -8,46 +8,64 @@ Let's take a look at a very basic API using Flask-Rebar:
 
 .. code-block:: python
 
+   # todos.py
+
    from flask import Flask
    from flask_rebar import Rebar
-   from flask_rebar import ResponseSchema
-   from marshmallow import fields
+   from marshmallow import Schema, fields
 
 
    rebar = Rebar()
    registry = rebar.create_handler_registry()
 
 
-   class TodoSchema(ResponseSchema):
-       id = fields.Integer()
+   class TodoSchema(Schema):
+       # The task we have to do.
+       description = fields.String(required=True)
+
+       # The unique id for this Todo. Determined by whatever database we're using.
+       id = fields.Integer(
+           # We'll explain the dump_only=True shortly.
+           dump_only=True
+       )
 
 
    @registry.handles(
       rule='/todos/<id>',
       method='GET',
-      response_body_schema=TodoSchema()  # for versions <= 1.7.0, use marshal_schema
+      # Marshal bodies of 200 (default) responses using the TodoSchema.
+      response_body_schema=TodoSchema()
    )
    def get_todo(id):
-       ...
-       return {'id': id}
+       todo = _get_todo_or_404(id)  # Some helper function that queries our database.
+       # If we got here, it means we were able to find a Todo with the given id.
+       # As with Flask, a 200 response code is assumed when we don't say otherwise.
+       # Return a dictionary with the data for this Todo, which Rebar will marshal
+       # into the response body using our TodoSchema, as we specified above.
+       return {'id': todo.id, 'description': todo.description}
 
    app = Flask(__name__)
    rebar.init_app(app)
 
-   if __name__ == '__main__':
-       app.run()
```

Comment:
(`app.run()` has been deprecated/discouraged for [several years](https://stackoverflow.com/a/39088836), and it's just as easy to show best practice here instead, so I fixed that too while I was in here.)